scripts/docker-machine-port-forwarding.sh

```diff
@@ -4,7 +4,8 @@
 
 function forward_port_if_not_forwarded {
     port=$1
-    forward_port_command="ssh -f -N -L $port:localhost:$port docker@$(docker-machine ip $(docker-machine active))"
+    machine=$(docker-machine active)
+    forward_port_command="ssh -f -N -i ~/.docker/machine/machines/$machine/id_rsa -L $port:localhost:$port docker@$(docker-machine ip $machine)"
```

Comment:
Using `~` in the path breaks the conditional that checks for an existing port forward. Can you change it to use `$HOME` instead? I.e:

```bash
forward_port_command="ssh -f -N -i $HOME/.docker/machine/machines/$machine/id_rsa -L $port:localhost:$port docker@$(docker-machine ip $machine)"
```